test/parse-iiif-manifest_test.js

```diff
@@ -0,0 +1,68 @@
+import Diva from '../source/js/diva';
+let v2Manifest = require('./manifests/iiifv2.json');
```

Comment:
Why do you load them like this?

Reply:
So that I could use them in the `index.html` as object data as well (was doing this while determining how to write the unit tests and for demo purposes)

I removed the `export default` from the files